code/modules/hippie/pool/pool_controller.dm

```diff
@@ -0,0 +1,363 @@
+
+//Originally stolen from paradise. Credits to tigercat2000.
+//Modified a lot by Kokojo.
+/obj/machinery/poolcontroller
+	name = "Pool Controller"
+	desc = "A controller for the nearby pool."
+	icon = 'icons/turf/pool.dmi'
```

Comment:
I would suggest putting this into the `hippie` icons folder, makes future rebases much more painless